testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -1395,7 +1394,7 @@ public void testMergeDeleteWithColumnMapping(String mode)
 
     @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS}, dataProvider = "columnMappingDataProvider")
     @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
-    public void testUnsupportedColumnMappingModeChangeDataFeed(String mode)
+    public void testUnsupportedColumnMappingModeChangeDataFeed(String mappingMode)
```

Comment:
The test name contains "Unsupported". Can we remove this test? I suppose TestDeltaLakeChangeDataFeedCompatibility already cover the column mapping mode with CDF.